spec/prog/postgres/postgres_server_nexus_spec.rb

```diff
@@ -3,58 +3,24 @@
 require_relative "../../model/spec_helper"
 
 RSpec.describe Prog::Postgres::PostgresServerNexus do
-  subject(:nx) { described_class.new(st) }
-
-  let(:st) { Strand.create(id: "0d77964d-c416-8edb-9237-7e7dd5d6fcf8", prog: "Postgres::PostgresServerNexus", label: "start") }
-
-  let(:postgres_server) {
-    instance_double(
-      PostgresServer,
-      id: "0d77964d-c416-8edb-9237-7e7dd5d6fcf8",
-      ubid: "pgubid",
-      version: "16",
-      timeline: instance_double(
-        PostgresTimeline,
-        id: "f6644aae-9759-8ada-9aef-9b6cfccdc167",
-        generate_walg_config: "walg config",
-        blob_storage: instance_double(MinioCluster, root_certs: "certs"),
-        aws?: false
-      ),
-      vm: instance_double(
-        Vm,
-        id: "1c7d59ee-8d46-8374-9553-6144490ecec5",
-        sshable: sshable,
-        ephemeral_net4: "1.1.1.1",
-        private_subnets: [instance_double(PrivateSubnet)]
-      ),
-      storage_device_paths: ["/dev/vdb"]
-    )
-  }
+  subject(:nx) { described_class.new(server_strand) }
+
+  let(:user_project) { Project.create(name: "test-project") }
+  let(:postgres_project) { Project.create(name: "postgres-service") }
 
-  let(:resource) {
-    instance_double(
-      PostgresResource,
-      ubid: "pgresourcesubid",
-      root_cert_1: "root_cert_1",
-      root_cert_2: "root_cert_2",
-      server_cert: "server_cert",
-      server_cert_key: "server_cert_key",
-      superuser_password: "dummy-password",
-      version: "16",
-      target_version: "16",
-      representative_server: postgres_server,
-      metric_destinations: [instance_double(PostgresMetricDestination, ubid: "pgmetricubid", url: "url", username: "username", password: "password")],
-      ca_certificates: "root_cert_1\nroot_cert_2",
-      location_id: Location::HETZNER_FSN1_ID,
-      project: instance_double(Project, get_ff_aws_cloudwatch_logs: true)
-    )
+  let(:postgres_resource) {
+    Prog::Postgres::PostgresResourceNexus.assemble(
+      project_id: user_project.id, location_id: Location::HETZNER_FSN1_ID,
+      name: "test-pg", target_vm_size: "standard-2", target_storage_size_gib: 64
+    ).subject
   }
 
-  let(:sshable) { Sshable.new }
+  let(:server_strand) { postgres_resource.servers.first.strand }
```

Comment:
Unless it is different:

```suggestion
  let(:server_strand) { postgres_server.strand }
```